### DISCUSSION TITLE
Status Bericht Iteration 25

### DISCUSSION THREAD

**Discussion by CodeRider101** [SE-TINF22B2/G2-Schoolify#278] [Status Berichte]
### ✅Was haben wir diese Woche geschafft?
- #256 
- #241 
- #276 
- #279 
- #241 
- insg 10h

### 🏆Was sind unsere Ziele für diese Woche und ggf. kommenden Wochen?
- #254 
- #253
- #251 
- #250
- #221 
- insg bereits: 3h 51min

### 🛠️Wie sind wir vorgegangen und gibt es Verbesserungsvorschläge?
Meeting am 01.05 und dort Updates und News besprochen. Zusätzlich alle gemeinsam den Projektscope durchgegangen und User Stories in die Labels 'Scope', bzw. 'NoScope' einkategorisiert.
Verbesserungsvorschläge:
\- keine -
### ⏱️Zeitaufwände (Abgeschlossen und angefangen)</p>
Name | Requirements | Design | Construction | Test | Project Management | Sonstiges | Sum [h]
-- | -- | -- | -- | -- | -- | -- | --
@laralngn | 3h 3min | - | - | - | - | - | 3,1
@Schmarvinius | - | - | - | - | - | - | -
@two-Gee | 3h 35min | - | - | - | - | - | 3,6
@Kronprinz03 | 37min | - | - | - | - | - | 0,6
@tilwbr | - | - | - | - | - | - | -
@CodeRider101 | - | - | - | - | 1h | - | 1
Alle | ProjektScope 336min | - | - | - | - | - | 5,6
Sum | - | - | - | - | - | - | 13,9

**Comment by CodeRider101** [comment]
Zeiten ergänze ich später

**Comment by Kronprinz03** [comment]
Danke dir 👍  finde es besser so wenn wir die genauen Zeiten haben diese auch hinzuschreiben :)

**Reply by CodeRider101** [reply]
Finde ich auch :) Danke dir für den Verbesserungsvorschlag
